OldTests/src/test/java/com/tle/webtests/test/workflow/rejection/NewVersionTest.java

```diff
@@ -15,81 +16,111 @@
-		ModerateListSearchResults modResults = taskListPage.exactQuery(itemFullName);
-		assertEquals(modResults.getStepName(itemFullName), "moderation step 1");
-		modResults.moderate(itemFullName).accept();
+		//login as admin to validate that we can't create a new version while an item is still under moderation
+		logon(ADMIN_USERNAME, BACKTICK_PASSWORD);
+		Boolean alertDisplayed = false;
+		try {
+			SearchPage.searchAndView(context, itemFullName).adminTab().newVersion();
+		}
+		catch(UnhandledAlertException f){
+			Alert alert = getContext().getDriver().switchTo().alert();
+			alert.accept();
+			alertDisplayed = true;
+		}
+		assertTrue( alertDisplayed, "Failed to trigger check of new version not allowed when already version in moderation.");
 		logout();
 
-		// login as the owner of the item and new version it.
-		logon("admin", "``````");
+		// Login as the SecondStepModerator and moderate again because this workflow needs moderating twice.
+		logon(SECOND_MODERATOR_USERNAME, BACKTICK_PASSWORD);
+		doModeration( SECOND_MODERATOR_USERNAME, BACKTICK_PASSWORD, "moderation step 2", itemFullName, false );
+		logout();
 
+		// Now the item should be in a state where we're allowed to create a new version.
+		// login as admin and new version it.
+		logon(ADMIN_USERNAME, BACKTICK_PASSWORD);
 		wizard = SearchPage.searchAndView(context, itemFullName).adminTab().newVersion();
-		wizard.editbox(1, itemFullName2);
-		wizard.save().submit();
-
-		// check that there is a version that is live, and one that is
-		// moderating.
-		ItemAdminPage adminPage = new ItemAdminPage(context).load();
-		adminPage.exactQuery(itemFullName);
-		adminPage.setSort("datemodified");
-		ItemListPage itemList = adminPage.results();
+		saveItem(wizard,1,itemFullName2);
+		// check that there is a version that is live, and one that is moderating.
+		ItemListPage itemList = getItemList(itemFullName);
 		assertEquals(itemList.getResultForTitle(itemFullName, 1).getStatus(), "live");
 		assertEquals(itemList.getResultForTitle(itemFullName2, 1).getStatus(), "moderating");
 		logout();
 
-		// Login as SimpleModerator and moderate the item at the first step.
-		logon("SimpleModerator", "``````");
-
-		taskListPage = new TaskListPage(context).load();
-		modResults = taskListPage.exactQuery(itemFullName2);
-		assertEquals(modResults.getStepName(itemFullName2), "moderation step 1");
-		modResults.moderate(itemFullName2).accept();
-
-		// After moderation, check that the item has gone life after the first
-		// moderation step
-		adminPage = new ItemAdminPage(context).load();
-		adminPage.exactQuery(itemFullName2);
-		adminPage.setSort("datemodified");
-		itemList = adminPage.results();
+		// Login as SimpleModerator and moderate the new version of this item at the first step.
+		logon(FIRST_MODERATOR_USERNAME, BACKTICK_PASSWORD);
+		doModeration(FIRST_MODERATOR_USERNAME, BACKTICK_PASSWORD, "moderation step 1", itemFullName2,false);
```

Comment:
Maybe create a constant for `"moderation step 1"` string, as it is used more than once. Probably step 2 as well, for consistency.